### PR TITLE
Updates template css file by removing the vendor specific prefixes and uses autoprefixer insteader

### PR DESCRIPTION
**Pull request by designerneil**
## Ticket(s)

**Jira Ticket**
- [EWL-4542: SG2 | Add autoprefixer to gulp](https://issues.ama-assn.org/browse/EWL-4542)

## Description
Update gulp to include grid vendor prefixes and remove vendor specific classes in template files

## To Test
- [ ] `gulp serve`
- [ ] visit http://localhost:3000/?p=templates-three-column
- [ ] inspect the page with Chrome inspector
- [ ] observe the class "ama__layout--three-column container " has both a-ms-grid-columns vendor prefix

## Visual Regressions
run `gulp test`

## Relevant Screenshots/GIFs
screen shot 2018-02-19 at 1 42 39 pm

## Remaining Tasks
N/A

## Additional Notes
N/A
---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

**Comment by designerneil** [comment]
https://github.com/AmericanMedicalAssociation/ama-style-guide-2/pull/114

File: styleguide/source/assets/scss/04-templates/_three_column.scss [hunk 1]
```diff
@@ -42,17 +36,12 @@
       grid-row-end: 4;
       grid-column: 1;
       -ms-grid-row: 2;
```
Comment (nstriedinger) [inline styleguide/source/assets/scss/04-templates/_three_column.scss]: Did you miss one here?

Comment (nstriedinger) [reply styleguide/source/assets/scss/04-templates/_three_column.scss]: @designerneil -- Are all the `-ms-` supposed to be removed?

Comment (designerneil) [reply styleguide/source/assets/scss/04-templates/_three_column.scss]: good catch.

**Pull request merged**